README.md

```diff
@@ -2,7 +2,8 @@
 
 ## Instructions
 
-This repository contains challenges you to undertake around object oriented programming in Java.
+This repository contains challenges you to undertake around object oriented programming in Java. You can work on these
```

Comment:
possible small type "for you to undertake"

Reply:
Hi @eggsy84 - this is now resolved :)